src/js/utilities.js

```diff
@@ -0,0 +1,91 @@
+const appendChild = (html, place) => {
+  const element = document.createRange().createContextualFragment(html);
+  document.querySelector(place).appendChild(element);
+};
+
+const store = (data, name) => {
+  localStorage.setItem(name, JSON.stringify(data));
+};
+
+const getFromStorage = (name) => {
+  const storage = JSON.parse(localStorage.getItem(name));
+  const elements = [];
+  if (storage) storage.forEach((element) => elements.push(element));
+  return elements;
+};
```

Comment:
Instead of using a forEach loop here, you could directly assign your `elements` = `storage` as this last one is your desired array of objects, but make sure it's not empty! you could do something like this:
```javascript
let elements;
if (!storage){ // If the storage if empty:
   elements = []; // Assign an empty array,
} else { // Else:
   elements = storage; // Assign the array of objects from LocalStorage.
}
```

Reply:
`let elements = storage || [];`

Reply:
Changes applied.